settings.php

```diff
@@ -165,6 +165,54 @@
             get_string('spmetadatasign_help', 'auth_saml2'),
             0, $yesno));
 
+    $settings->add(new admin_setting_configtext(
+        'auth_saml2/entityid',
+        get_string('entityid', 'auth_saml2'),
+        get_string('entityid_help', 'auth_saml2'),
+        "$CFG->wwwroot/auth/saml2/sp/metadata.php"
+    ));
+
+    $settings->add(new admin_setting_configselect(
+        'auth_saml2/wantassertionssigned',
+        get_string('wantassertionssigned', 'auth_saml2'),
+        get_string('wantassertionssigned_help', 'auth_saml2'),
+        0, $yesno
+    ));
+
+    $assertionsconsumerservices = [
+        'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST' => 'HTTP Post',
+        'urn:oasis:names:tc:SAML:1.0:profiles:browser-post' => 'Browser post profile',
+        'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Artifact' => 'HTTP Artifact',
+        'urn:oasis:names:tc:SAML:1.0:profiles:artifact-01' => 'Artifact 01 profile',
+    ];
+
+    $settings->add(new admin_setting_configmultiselect(
+        'auth_saml2/assertionsconsumerservices',
+        get_string('assertionsconsumerservices', 'auth_saml2'),
+        get_string('assertionsconsumerservices_help', 'auth_saml2'),
+        [
```

Comment:
worth a micro refactor of array_keys($assertionsconsumerservices) ?

Reply:
yep, done